specification/schemas/resources/Practitioner.yaml

```diff
@@ -10,21 +10,8 @@ properties:
     type: array
     description: The prescriber's name.
     items:
-      type: object
-      properties:
-        family:
-          type: string
-          description: Family name (often called Surname).
-          example: Toboggan
-        given:
-          type: array
-          description: Given names, including any middle names.
-          items:
-            type: string
-            example: Mantis
-        prefix:
-          type: array
-          description: Name prefixes, titles, and prenominals.
-          items:
-            type: string
-            example: Dr
+      oneOf:
+        - $ref: ../fragments/structuredName.yaml
+        - type: string
```

Comment:
I don't think it's a string directly in the array is it? I thought it was an object with one property called "text"